pkg/minikube/machine/cache_images.go

```diff
@@ -314,10 +314,19 @@ func CacheImage(image, dst string) error {
 	}
 
 	glog.Infoln("OPENING: ", dstPath)
-	f, err := os.Create(dstPath)
+	tmpPath := dstPath + ".tmp"
+	f, err := os.Create(tmpPath)
```

Comment:
Will this fail if dstPath.tmp exists already? 

I'd personally rather see ioutil.TempFile used here, passing in this same directory as an argument.

Reply:
It should truncate, but we can use random file names